tests/integration_tests/test_autopilot_job_status_polling_udf_real.py

```diff
@@ -42,17 +43,23 @@ def get_emitted(self):
 @pytest.mark.skipif(not aws_params.aws_access_key,
                     reason="AWS credentials are not set")
 def test_autopilot_training_status_udf_real():
+
+    for setup_params in [reg_setup_params, cls_setup_params]:
```

Comment:
better use parameterized tests here, or have two test which call the run_test with different parameter, because with for loop you run two tests in on test and if the first fails the second doesn't run